Unicode support for interactive setup

I got this error when trying to generate a new probot app:

```console
$ npx create-probot-app we-are-on-gitlab
npx: installed 229 in 21.396s

Let's create a Probot app!
Hit enter to accept the suggestion.

? App name: we-are-on-gitlab
? Description of app: Tell that development happens on GitLab in new issue or pull requests.
? Author's full name: Timothée Mazzucotelli
? Author's email address: pawamoy@pm.me
? GitHub user or org name: pawamoy
? Repository name: we-are-on-gitlab
? Which template would you like to use? basic-js
created file: /home/pawamoy/we-are-on-gitlab/.env.example
created file: /home/pawamoy/we-are-on-gitlab/CODE_OF_CONDUCT.md
created file: /home/pawamoy/we-are-on-gitlab/CONTRIBUTING.md
created file: /home/pawamoy/we-are-on-gitlab/LICENSE
created file: /home/pawamoy/we-are-on-gitlab/README.md
created file: /home/pawamoy/we-are-on-gitlab/app.yml
created file: /home/pawamoy/we-are-on-gitlab/.gitignore
created file: /home/pawamoy/we-are-on-gitlab/index.js
created file: /home/pawamoy/we-are-on-gitlab/package.json
created file: /home/pawamoy/we-are-on-gitlab/test/index.test.js
created file: /home/pawamoy/we-are-on-gitlab/test/fixtures/issues.opened.json
created file: /home/pawamoy/we-are-on-gitlab/test/fixtures/mock-cert.pem

Finished scaffolding files!

Initialized a Git repository.

Installing dependencies. This may take a few minutes...

npm ERR! code EJSONPARSE
npm ERR! file /home/pawamoy/we-are-on-gitlab/package.json
npm ERR! JSON.parse Failed to parse json
npm ERR! JSON.parse Unexpected token x in JSON at position 184 while parsing near '...  "author": "Timoth\xE9e Mazzucotelli <p...'
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pawamoy/.npm/_logs/2019-11-29T17_50_15_194Z-debug.log
```

Problem seems to be the `é` in my name, being written as `\xE9` in `package.json`.

--- a/script/test
+++ b/script/test
@@ -42,7 +42,7 @@ do
   "$PROJECT_DIR"/bin/create-probot-app.js \
     --appName "travis-template-test" \
     --desc "A Probot App for building on Travis" \
-    --author "Pro Bawt" \
+    --author "Pro Báwt" \
     --email "probot@example.com" \
     --homepage "https://probot.github.io" \
     --user "probot" \

--- a/src/create-probot-app.ts
+++ b/src/create-probot-app.ts
@@ -37,6 +37,7 @@ function sanitizeBy(object: {
   keys.forEach(key => {
     if (key in object) {
       object[key] = jsesc(object[key], {
+        minimal: true,
         quotes: 'double'
       })
     }

